packages/react-admin/src/mui/list/SingleFieldList.spec.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+
+import { SingleFieldList } from './SingleFieldList';
+import ChipField from '../field/ChipField';
+
+describe('<SingleFieldList />', () => {
+    it('should render a link to the Edit page of the related record by default', () => {
+        const wrapper = shallow(
+            <SingleFieldList
+                ids={[1, 2]}
+                data={[{ id: 1, title: 'foo' }, { id: 2, title: 'bar' }]}
+                resource="bar"
+                basePath="/bar"
```

Comment:
I don't understand this test case. Is it about a reference many of `bar` to `bar`? I suggest you use `post` and `author` or `comment` to make more sense of it.

Reply:
The `SingleFieldList` receive the referenced resource as `resource` and the related `basePath`. It does not have knowledge about the original resource which reference its own resource.